client/my-sites/pages/page-list.jsx

```diff
@@ -299,7 +299,7 @@ class Pages extends Component {
 		return (
 			<SectionHeader label={ translate( 'Pages' ) }>
 				<Button primary compact className="pages__add-page" href={ newPageLink }>
-					{ translate( 'Add New Page' ) }
+					{ translate( 'Add new page' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.